packages/common/src/pages/_internal/PageBusinessRules.ts

```diff
@@ -60,34 +70,34 @@ export const PagePermissions = [
  * @private
  */
 export const PagePermissionPolicies: IPermissionPolicy[] = [
+  {
+    permission: "hub:page",
+    services: ["portal"],
+    licenses: ["hub-basic", "hub-premium", "enterprise-sites"],
+  },
   {
     permission: "hub:page:create",
-    subsystems: ["pages", "sites"],
+    dependencies: ["hub:page"],
     authenticated: true,
     privileges: ["portal:user:createItem"],
-    licenses: ["hub-basic", "hub-premium", "enterprise-sites"],
   },
   {
     permission: "hub:page:view",
-    subsystems: ["pages", "sites"],
+    services: ["portal"],
```

Comment:
Wouldn't this permission be better expressed by removing the service and licenses and adding `dependencies: ["hub:page"]`?

Reply:
Yes. Moving fast, copying from other things where the base check has a license that we don't want applied to the :view - thanks for catching this :)